### PR TITLE
moosefs: update to 3.0.117

### PR DESCRIPTION
**Pull request by toomyem**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x64-glibc)

### CODE REVIEW

**Comment by toomyem** [comment]
Sorry. I've made PR from master branch of my fork by mistake. The only way to correct this was to create new PR. It won't happen again.

**Comment by mhmdanas** [comment]
No worries. :)

File: srcpkgs/moosefs/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'moosefs'
 pkgname=moosefs
-version=3.0.113
+version=3.0.117
 revision=2
```
Comment (mhmdanas) [inline srcpkgs/moosefs/template:4]: Revision should be reset to 1 when bumping version.

File: srcpkgs/moosefs/template [hunk 2]
```diff
@@ -12,7 +12,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="http://moosefs.org/"
 distfiles="http://ppa.moosefs.com/src/moosefs-${version}-1.tar.gz"
```
Comment (mhmdanas) [inline srcpkgs/moosefs/template:14]: Those URLs should work with HTTPS too.

File: srcpkgs/moosefs/template [hunk 3]
```diff
@@ -12,7 +12,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
...
-checksum=f88cc0142b5fe8a6a4c3fcb2ae58053fcb98787848cf09286a57718a982093a9
+checksum=d77947d0d8b699e2373926a1274ae81a9a8b24445c5ab986a1ec5e9203e3c3ba
 python_version=2 #unverified
```
Comment (mhmdanas) [inline srcpkgs/moosefs/template:16]: Upstream apparently supports Python 3 too, can you try that?

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: srcpkgs/moosefs/template [hunk 4]
```diff
@@ -10,10 +10,10 @@ makedepends="zlib-devel libpcap-devel fuse-devel"
 short_desc="Fault tolerant, network distributed file system"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
-homepage="http://moosefs.org/"
-distfiles="http://ppa.moosefs.com/src/moosefs-${version}-1.tar.gz"
-checksum=f88cc0142b5fe8a6a4c3fcb2ae58053fcb98787848cf09286a57718a982093a9
-python_version=2 #unverified
+homepage="https://moosefs.org/"
```
Comment (classabbyamp) [inline srcpkgs/moosefs/template:13]:
```suggestion
homepage="https://moosefs.com/"
```

**Pull request merged**